source/globalCommands.py

```diff
@@ -3458,7 +3459,11 @@ def script_reportClipboardText(self,gesture):
 		else:
 			# Translators: If the number of characters on the clipboard is greater than about 1000, it reports this message and gives number of characters on the clipboard.
 			# Example output: The clipboard contains a large portion of text. It is 2300 characters long.
-			ui.message(_("The clipboard contains a large portion of text. It is %s characters long") % len(text))
+			ui.message(ngettext(
+				"The clipboard contains a large portion of text. It is %s character long",
+				"The clipboard contains a large portion of text. It is %s characters long",

```

Comment:
"portion" is a weird word here. "quantity" or "amount" would be preferable.

```suggestion
				"The clipboard contains a large amount of text. It is %s characters long",
```

Or, less formal: "... contains a lot of text ..."